File: Models/KerasTF/dense.py
import tensorflow as tf
from tensorflow import keras
from tensorflow.keras import layers


def DenseModel():

    model = keras.Sequential()
    model.add(keras.layers.Flatten(input_shape=(28, 28)))
    model.add(layers.Dense(128, activation="relu", name="layer1"))
    model.add(layers.Dense(10, activation="relu", name="layer2"))
    model.add(layers.Dense(10, name="layer3"))
    return model

File: Models/Swift4TF/Dense.swift
import TensorFlow


public struct DenseModel: Layer {
    var layer1, layer2, layer3: Dense<Float> 

    public init() {
        layer1 = Dense<Float>(inputSize: 784, outputSize: 10, activation: relu)
        layer2 = Dense<Float>(inputSize: 10, outputSize: 10, activation: relu)
        layer3 = Dense<Float>(inputSize: 10, outputSize: 10)
    }

    @differentiable
    public func callAsFunction(_ input: Tensor<Float>) -> Tensor<Float> {
        return input.sequenced(through: layer1, layer2, layer3)
    }
}
